UN-OCHA/pocam-site
# POCAM-30 update ocha_basic theme

I cloned the ocha_basic theme, renamed it to ochabasic and delete its git folder

### Summary
1. consolidate the colours, and replace with OCHA brand colours where relevant
2. combine and remove duplicate variables
3. new folder for pocam_custom where I placed certain sass files from the current theme that have a lot of specific custom rules
4. include the POCAM logo (currently 2 versions on mobile)
5. variant for site-menu called site-menu--compact with its own include and scss
6. override file for main menu scss to disable to mobile menu toggle. It displays a single menu item no matter the viewport width
7. copy specific template overrides and preprocess functions from current theme

### Features
Remove theme menu item from user menu
New `global` context for main menu block placement (and other CD related blocks in the future)

### How to test
I'm hoping this just works :)
1. Checkout branch
2. Features revert
3. Clear cache
4. Check the front end for anything obviously not like how it used to be (more details here to come)

### To do

1. Regroup and get input from others
2. Grunt criticalCSS to gulp
3. logo discussion and gather dimension for homescreen icons and favicons to help designer prep
